inline-c/src/Language/C/Types/Parse.hs

```diff
@@ -212,13 +215,14 @@ quickCParser typeNames s p = case runCParser typeNames "quickCParser" s p of
 -- | Like 'quickCParser', but uses @'cCParserContext' ('const' 'False')@ as
 -- 'CParserContext'.
 quickCParser_
-  :: String
+  :: Bool
```

Comment:
cCParserContext has The Bool to change the parser for C++.
cxxidentifier_no_lex is not exported, because identifier_no_lex reads and uses  CParserContext' value to change the parser.